Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1123565
Upstream-status: Send upstream for 3.17 with Cc: stable
Note: Not needed for 3.16 and higher since use_native_backlight=1 is the
default there, upstream is maintaining the quirk list for now in case it is
decided to flip the default back.

From e1eaa90a9691696df34040f40f5dbc1d91a394f0 Mon Sep 17 00:00:00 2001
From: Hans de Goede <hdegoede@redhat.com>
Date: Mon, 28 Jul 2014 17:31:44 +0200
Subject: [PATCH] acpi-video: Add video.use_native_backlight=1 for HP EliteBook
 2014 models

https://bugzilla.redhat.com/show_bug.cgi?id=1123565

Cc: stable@vger.kernel.org
Signed-off-by: Hans de Goede <hdegoede@redhat.com>
---
 drivers/acpi/video.c | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/drivers/acpi/video.c b/drivers/acpi/video.c
index 18c0e69..8268843 100644
--- a/drivers/acpi/video.c
+++ b/drivers/acpi/video.c
@@ -673,6 +673,15 @@ static struct dmi_system_id video_dmi_table[] __initdata = {
 	},
 	{
 	.callback = video_set_use_native_backlight,
+	.ident = "HP EliteBook 2014 models",
+	.matches = {
+		DMI_MATCH(DMI_SYS_VENDOR, "Hewlett-Packard"),
+		DMI_MATCH(DMI_PRODUCT_NAME, "HP EliteBook "),
+		DMI_MATCH(DMI_PRODUCT_NAME, " G2"),
+		},
+	},
+	{
+	.callback = video_set_use_native_backlight,
 	.ident = "HP ZBook 14",
 	.matches = {
 		DMI_MATCH(DMI_SYS_VENDOR, "Hewlett-Packard"),
-- 
2.0.1

